services/ui-src/src/routes/McparReportPage/McparReportPage.tsx

```diff
@@ -25,10 +32,37 @@ export const McparReportPage = ({ pageJson }: Props) => {
   const previousRoute = findRoute(mcparRoutes, path, "previous", "/mcpar");
   const nextRoute = findRoute(mcparRoutes, path, "next", "/mcpar");
 
-  const onSubmit = () => {
-    // TODO: Wire up submit functionality
+  // get user's state
+  const { user } = useUser();
+  const { state, userRole } = user ?? {};
+  const reportYear = "2022";
+  const reportKey = `${state}${reportYear}`;
+
+  // TODO: get real program name per report
+  const programName = "tempName";
```

Comment:
Is this still needed?

Reply:
Yes this will be added later when users have the ability to enter report names. For now it'll just use this